contracts/HatchOracle.sol

```diff
@@ -56,22 +55,20 @@ contract HatchOracle is AragonApp, IACLOracle {
 
     /**
      * @notice Update ratio to `_ratioNom` / `_ratioDen`
-     * @param _ratioNom The new nominator for ratio between sent tokens and token balance
-     * @param _ratioDen The new denominator for ratio between sent tokens and token balance
+     * @param _ratio The new ratio between sent tokens and token balance
      */
-    function setRatio(uint256 _ratioNom, uint256 _ratioDen) external auth(SET_RATIO_ROLE) {
-        ratioNom = _ratioNom;
-        ratioDen = _ratioDen;
+    function setRatio(uint256 _ratio) external auth(SET_RATIO_ROLE) {
+        ratio = _ratio;
 
-        emit RatioSet(_ratioNom, _ratioDen);
+        emit RatioSet(_ratio, PPM);
```

Comment:
You can use `RatioSet(_ratio);` if in line 29 you have `event RatioSet(uint256 ratio)`.